src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,33 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">
+        {post.title}
+      </h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={post.user} />
```

Comment:
Let's think about what will be if there is no user. Add a conditional rendering here

Reply:
Пост без юзера? А хто його створив? Дух божий?

Reply:
Oh, in the App.tsx you use the function `getUserById` which finds a user from another array and **may** return `null`.

![image](https://github.com/mate-academy/react_static-list-of-posts-js/assets/86923089/d8bd317e-237e-4e9b-bccd-5f751dfbfce6)

So it doesn't matter what is it - you **must** handle all theoretical variants to avoid possible unhandled errors